FreeRTOS-Plus/Demo/Common/Logging/windows/Logging_WinSim.c

```diff
@@ -71,6 +71,11 @@
 /* A block time of zero simply means don't block. */
 #define dlDONT_BLOCK                    0
 
+/* Choose LF as the default new-line character. */
+#ifndef configLOGGING_NEW_LINE
+    #define configLOGGING_NEW_LINE    "\n"
```

Comment:
As the file is called Logging_WinSim - shouldn't the default be "\r\n"?

Reply:
Looks like if the above were the case the other changes would not be required.  Although taking out the task pool constants is desirable.

Reply:
Agreed. As this is used by other WinSim demos besides MQTT, I was a little worried about them using '\n' as the default. Upon further inspection, this was only true for `FreeRTOS_Plus_TCP_Minimal_Windows_Simulator`, so I updated that demo to use CRLF instead and have set the default to CRLF as well.